plugins/woocommerce/changelog/add-53008_add_to_cart_wrapper_block

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Create Add to Cart With Options wrapper block #53215
```

Comment:
Should this be a comment instead of a changelog entry, considering it's an experimental block?

Reply:
Done in [480903e](https://github.com/woocommerce/woocommerce/pull/53215/commits/480903e6247b3da21814503b03082d2ba726fc8f)